python/hail/methods/qc.py

```diff
@@ -169,3 +169,512 @@ def variant_qc(dataset, name='variant_qc'):
     """
 
     return MatrixTable(Env.hail().methods.VariantQC.apply(dataset._jvds, 'va.`{}`'.format(name)))
+
+
+@handle_py4j
+@typecheck(dataset=MatrixTable,
+           config=strlike,
+           block_size=integral,
+           name=strlike,
+           csq=bool)
+def vep(dataset, config, block_size=1000, name='vep', csq=False):
+    """Annotate variants with VEP.
+
+    .. include:: ../_templates/req_tvariant.rst
+
+    :py:meth:`~hail.methods.vep` runs `Variant Effect Predictor
+    <http://www.ensembl.org/info/docs/tools/vep/index.html>`__ with the `LOFTEE
+    plugin <https://github.com/konradjk/loftee>`__ on the current dataset and
+    adds the result as a row field.
+
+    Examples
+    --------
+
+    Add VEP annotations to the dataset:
+
+    >>> result = methods.vep(dataset, "data/vep.properties") # doctest: +SKIP
+
+    Notes
+    -----
+
+    **Configuration**
+
+    :py:meth:`~hail.VariantDataset.vep` needs a configuration file to tell it
+    how to run VEP. The format is a `.properties file
+    <https://en.wikipedia.org/wiki/.properties>`__. Roughly, each line defines a
+    property as a key-value pair of the form `key = value`. `vep` supports the
```

Comment:
Should these both be double-backticks so as to render fixed-width? I feel more strongly about the name of binary executable being fixed-width.